client/game.cpp

```diff
@@ -129,6 +129,18 @@ void Game::Update(double delta, double mouse_x, double mouse_y, int window_width
 	rs.Update(delta, client_state);
 	lua_sys.Update(delta);
 
+	// Franes per second
+	//
+	// If one second has passed, we have the number of frames in that second, giving us frames per second
+	frame_deltas += delta;
+	frames++;
+	if (frame_deltas >= 1) {
+		fps = frames;
+		frame_deltas = 0;
+		frames = 0;
+		//std::cout << std::to_string(fps) << std::endl;
```

Comment:
Debug line?

Reply:
initially just went to console, removing